doc/source/api-reference/qibo.rst

```diff
@@ -1128,22 +1128,26 @@ The quantum errors available to build a noise model are the following:
     :member-order: bysource
 
 
-Realistic noise model
-^^^^^^^^^^^^^^^^^^^^^
+IBMQ noise model
+^^^^^^^^^^^^^^^^
 
-In Qibo, it is possible to build a realistic noise model of a real quantum computer
-by using the :meth:`qibo.noise.NoiseModel.composite()` method.
+In Qibo, it is possible to build noisy circuits based on IBMQ's reported noise model of
+for its quantum computer by using the :class:`qibo.noise.IBMQNoiseModel` class.
```

Comment:
I think there might be a typo here in "reported noise model of for its quantum computer".